explanations/best-practices/code-reviews.rst

```diff
@@ -0,0 +1,41 @@
+============
+Code reviews
+============
+
+Writing code is a craft. There is no absolute right or wrong way of expressing thoughts into code. It's mainly contextual:
+
+- What the current needs?
+- What are the current constraints?
+- What is our current understanding of the problem?
+
+Reviewing code is also contextual. Here below are some guidelines. They must not be applied in every circumstance. Instead, they define a baseline in terms of expectations on the team.
+
+Requesting a review
+===================
+
+- **Timing:** expect the reviewer to come back after a day or two, in order to take timezones into account.
```

Comment:
I think this belongs in the `Performing a review` section.

I suggest we also be a bit more explicit about the expectations:

> * Reviewers should endeavour to reply within 24hrs of the request, and are expected to reply within 48hrs.

We could also consider adding:

> If a review can't be performed within this time frame, the reviewer should redirect to someone who is able or at the very least leave a comment informing the author of the delay.

Reply:
Okay. I moved it and tweaked your proposal notably to talk in business days instead of hours (to take weekends into account).